geopandas/base.py

```diff
@@ -618,6 +618,33 @@ def exterior(self):
         # TODO: return empty geometry for non-polygons
         return _delegate_property("exterior", self)
 
+    @property
```

Comment:
Any specific reason to make this property (also on an array level)?

Reply:
I wanted to check this with you actually, and what the preferred approach would be - looking at the current interface, Shapely methods that takes no additional parameters except the input data have been marked as a property e.g. [convex_hull](https://github.com/geopandas/geopandas/blob/979730b6fa1fa9a5f8102b6dcb99ec78a627954b/geopandas/array.py#L516)

Reply:
Yeah... I believe that was a mistake made years ago :). Property shall be something that is super cheap, not a method creating new geometry imo. So I'd rather not add more of them, given we already have inconsistent API due to this (e.g. centroid is a property but represetnative_point is a method...). Let's stick to methods for all these new shapely wrappers.

Reply:
That would be my preference - I'll get this updated. Do we need an issue to track updating the inconsistent API?

Reply:
That is already partially happening here #1616. We never properly discussed this and live with the inconsistency but it is on the [roadmap](https://geopandas.org/en/stable/about/roadmap.html#clarity-of-the-api).